WordPress/wporg-main-2022
# Add meta descriptions

Still needs to be updated for the new site.




Fixes #7.












### How to test the changes in this Pull Request:

1. Edit the front page post, and set the Excerpt to a known string.
2. Refresh the front page.
3. View source and confirm that your string is in the content attribute of both `<meta name="description">` and `<meta property="og:description">`
4. `yarn lighthouse` and confirm that the SEO score is 100.
